src/test/java/org/jenkinsci/plugins/codesonar/integration/conditions/WarningCountIncreaseSpecifiedScoreAndHigherConditionIT.java

```diff
@@ -24,7 +26,8 @@ public void percentageOfWarningsBellowTheDesignatedScoreIsAboveTheThreshold_Buil
         // arrange
         final Result EXPECTED_RESULT = Result.FAILURE;
         final String WARRANTED_RESULT = Result.FAILURE.toString();
-        final String VISIBILITY_FILTER = "2"; // active warnings
+        final String VISIBILITY_FILTER = "active"; // active warnings
+        final String VISIBILITY_FILTER_NEW_WARNINGS = "all"; // all warnings
```

Comment:
1. The comments "active warnings", "all warnings" are unnecessary and should be removed.
2. Should `VISIBILITY_FILTER_NEW_WARNINGS` be set to to "new" and not "all"?